### ISSUE TITLE
Display a short stage name in front of gate chips

### ISSUE THREAD

**Issue by jrobbins** [GoogleChrome/chromium-dashboard#2519]
**Is your feature request related to a problem? Please describe.**
We currently show gate chips on the myfeatures page in the pending-my-review box.  These gate chips clearly show the gate type (e.g., "API owners").  However, the stage is not shown until after the user clicks to see that information in the approvals dialog.  The stage name could be useful context, e.g., the API owners might want to focus their efforts on features that are seeking approval to ship before looking at features that are seeking approval for origin trials.  Also, it is possible for a feature owner to be seeking approval on multiple stages at the same time, so showing the stage name will disambiguate among multiple rows of gate chips (one row for each stage).  

**Describe the solution you'd like**
Basically, implement the TODO comment in chromedash-feature-row.js function renderActiveStageAndGates().

The layout is already done, so I think we just need a value for the variable `stageName`.  Also, some of the names of stages can be pretty long, so we should probably use one-word or two-word labels for the state names to keep the display compact.

**Additional context**
The chromedash-feature-table element fetches a gates for each feature, and then it passes that information to chromedash-feature-row.  However, in that data the stages are only represented by the stage_id numeric field of each gate.  We don't yet make a call to get the list of stages for each feature.

**Comment by DanielRyanSmith** [comment]
Addressed in #2634
